### PR TITLE
Implemented publication to Maven Central

### PR DESCRIPTION
**Pull request by shanshin**

### CODE REVIEW

**Review by qwwdfsad — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,35 +13,118 @@ and [JaCoCo](https://github.com/jacoco/jacoco).
...
-### Add repository
+### Add Maven Central repository
 
 For `settings.gradle`
 ```
 pluginManagement {
     repositories {
         // ... other dependencies
-        maven { url 'https://maven.pkg.jetbrains.space/public/p/jb-coverage/maven' }
+        mavenCentral()
```
Comment (sandwwraith) [inline README.md:23]: Isn't it included by default?

File: README.md [hunk 2]
```diff
@@ -13,35 +13,118 @@ and [JaCoCo](https://github.com/jacoco/jacoco).
...
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.jetbrains.kotlinx:kover:0.3.0'
+    }
 }
+  
+apply plugin: 'org.jetbrains.kotlinx.kover'
```
Comment (sandwwraith) [inline README.md:69]: Can we still apply it with `kotlinx-cover`, as with `apply plugin: 'binary-compatibility-validator'` ?

File: README.md [hunk 3]
```diff
@@ -13,35 +13,118 @@ and [JaCoCo](https://github.com/jacoco/jacoco).
...
+}
+```
+For `build.gradle`
+```
+plugins {
+    id 'org.jetbrains.kotlinx.kover' version '0.3.0' apply(false)
+}
+
+subprojects {
+    apply plugin: 'org.jetbrains.kotlinx.kover'
```
Comment (sandwwraith) [inline README.md:124]: This haven't worked for me for some strange reason

Comment (shanshin) [reply README.md:124]: Error?

File: README.md (hunk 2 above)
Comment (shanshin) [reply README.md:69]: I can add an additional plugin id `kover` to the resources for legacy scripts

**Pull request merged**